client/packages.go

```diff
@@ -24,12 +24,12 @@ import (
 	"fmt"
 )
 
-// Package represents a Snap package
-type Package struct {
+// Snap represents a Snap snap
```

Comment:
This is `client`, which is supposed to be completely separate from `snap` and only in the same project for convenience.

And changing server to daemon, for clarity. (although maybe "snapd" would be better)